### ISSUE TITLE
Minor bug with puppet-zookeeper/templates/zookeeper.service.erb

### ISSUE THREAD

**Issue by tamama** [deric/puppet-zookeeper#103]
Hi there,

1. There is a minor bug with `puppet-zookeeper/templates/zookeeper.service.erb`, highlighted in bold

```
[Unit]
  Want=
```
(should be **Wants**)

2.  The `classpath` should include `/usr/lib/zookeeper/lib/*:/usr/lib/zookeeper/zookeeper.jar`. Otherwise, `systemd` could not start `zookeeper-server.service`.

Thank you very much.

Kind regards,
Tamama

Here is the manifest file to reproduce the problem on any Centos/7 VM.

`site/profile/manifests/zookeeper.pp`
```puppet
class profile::zookeeper {
    notify { '[profile::zookeeper] Starting ... ' : }

    Package { provider => 'dnf' }
    
    class { 'zookeeper' :
        require                 => [ Class['::profile::base'] , Class['::profile::clouderarepo'] ] ,

        client_ip               => $::ipaddress_tun0 ,

        client_port             => 52181 ,
        election_port           => 52888 ,
        leader_port             => 53888 ,

        packages                => [ 'zookeeper' , 'zookeeper-server'] ,
        service_name            => 'zookeeper-server' ,
        initialize_datastore    => true ,

        service_provider        => 'systemd' ,
        manage_service_file     => true ,
        manage_service          => true ,
        systemd_unit_after      => 'network-online.target openvpn-client@.service' ,
        systemd_unit_want       => 'network-online.target openvpn-client@.service' ,

        java_opts               => '-cp "/usr/lib/zookeeper/lib/*:/usr/lib/zookeeper/zookeeper.jar"'
    }

    notify { '[profile::zookeeper] ... finished' : }
}
```

**Comment by deric** [comment]
@tamama Thanks for reporting this! 

Regarding the second issue, is it possible that [this line](https://github.com/deric/puppet-zookeeper/blob/master/templates/zookeeper.service.erb#L17):
```
CLASSPATH="$([ -r /usr/lib/zookeeper/bin/zkEnv.sh ] && . /usr/lib/zookeeper/bin/zkEnv.sh ; echo $CLASSPATH)";
```
overwrites correct path?

**Comment by tamama** [comment]
Hi Deric, 

You are most welcome.

Regarding your question, nope - that line does not override the correct path. (This conclusion is drawn from an installation on a clean VM). 
Please see the following for my reasons.

Kind regards,
Tamama



**Before that line** - (systemd logs - journalctl -xe -u zookeeper-server)
...
+ CLASSPATH='/etc/zookeeper/conf:/usr/share/java/jline.jar:/usr/share/java/log4j-1.2.jar:/usr/share/java/xercesImpl.jar:/usr/share/java/xmlParserAPIs.jar:/usr/share/java/netty.jar:/usr/share/java/slf4j-api.jar:/usr/share/java/slf4j-log4j12.jar:/usr/share/java/zookeeper.jar

...

**After that line** -
...
+ CLASSPATH='/usr/bin/../build/classes:/usr/bin/../build/lib/*.jar:/usr/bin/../lib/*.jar:/usr/bin/../zookeeper-*.jar:/usr/bin/../src/java/lib/*.jar:/etc/zookeeper/conf:/usr/share/java/jline.jar:/usr/share/java/log4j-1.2.jar:/usr/share/java/xercesImpl.jar:/usr/share/java/xmlParserAPIs.jar:/usr/share/java/netty.jar:/usr/share/java/slf4j-api.jar:/usr/share/java/slf4j-log4j12.jar:/usr/share/java/zookeeper.jar
...

I look into the systemd logs (journalctl -xe -u zookeeper-server) after re-installing puppet-zookeeper on a clean VM.

On a clean VM,

**Comment by deric** [comment]
Should be fixed in [v0.7.5 release](https://github.com/deric/puppet-zookeeper/releases/tag/v0.7.5).
